Composer/packages/lib/indexers/src/utils/lgUtil.ts

```diff
@@ -259,8 +259,8 @@ export function extractOptionByKey(nameOfKey: string, options: string[]): string
   return result;
 }
 
-export function parse(id: string, content: string, lgFiles: TextFile[]): LgFile {
-  const lgImportResolver = importResolverGenerator(lgFiles, '.lg');
+export function parse(id: string, content: string, lgResources: LGResource[]): LgFile {
```

Comment:
Can we create LGResource inside this methods, then for caller, needs less knowledge,  still pass lgFiles:{id, content}[]. I don't have strong opinions on this.